.travis.yml

```diff
@@ -115,8 +108,7 @@ install:
   # for python 3 we dont install everything yet..
   - |
       [ $TESTS != py3 ] || \
-      pip install -e slave \
```

Comment:
There is no such tests in master branch indeed. I have compatibility tests in #2136 , but I can add installation of `buidlbot-slave` in Travis there, when I will finish it.